src/shared/telemetry/telemetryLogger.ts

```diff
@@ -14,9 +14,9 @@ export interface MetricQuery {
     readonly metricName: string
 
     /**
-     * Attributes to filter out of the metadata
+     * Exclude metadata items matching these keys.
      */
-    readonly filters?: string[]
+    readonly excludeKeys?: string[]
```

Comment:
Renamed because "filters" is ambiguous: it can mean include or exclude. In `window.showSaveDialog()` it means "include" :)

Reply:
Yeah I think I originally had the field as a callback or something to align with `Array.filter`. Tbh I'm still confused by `filter` even after using it so much...